### PR TITLE
Schema code based on diagram

### PR DESCRIPTION
**Pull request by ulisesedu7**
## Description
This project is creating a schema SQL code based on a diagram. 

**Task:** https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md

This PR makes the following changes:
- Created the tables based on the diagram
- Add foreign keys to connect the tables
- Add a many-to-many relation table

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist

- [X] My code follows the style guidelines of this project (no linters errors)
- [X] I have checked my code and corrected any misspellings
- [X] I have commented on my code
- [X] I have deleted any unused code as comments in my codebase

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+ALTER TABLE invoices ADD CONSTRAINT fk_invoices FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id);
+
+-- Create Many to Many Relationship
+CREATE TABLE history_treatments (
+  id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
+  treatment_id INT,
+  medical_history_id INT,
+  PRIMARY KEY (treatment_id, medical_history_id)
+);
```
Comment (mateo951) [inline schema_based_on_diagram.sql:63]: All tables were created ✔

Comment (mateo951) [inline schema_based_on_diagram.sql:63]: Awesome! Many-to-many relationship is created ✔

Comment (mateo951) [reply schema_based_on_diagram.sql:63]: All tables have a PRIMAR KEY 🔥

Comment (mateo951) [reply schema_based_on_diagram.sql:63]: All FOREIGN KEYS were set 🐐 Well done!

**Review by mateo951 — APPROVED** [review]
### Hi @ulisesedu7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🤓 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ulisesedu7** [comment]
@mateo951 Hey Mateo, thank you very much for reviewing my code! I appreciate it a lot. Have a nice day!

**Pull request merged**